macosx/SparkleProxy.mm

```diff
@@ -1,4 +1,4 @@
-// This file Copyright © 2005-2022 Transmission authors and contributors.
+// This file Copyright © 2022-2022 Transmission authors and contributors.
```

Comment:
Again here, I don't think we should update copyright year

Reply:
This file and content was created by me this year.